topics/ecology/tutorials/de-novo-rad-seq/workflows/Galaxy-Workflow-de-novo-rad-seq.ga

```diff
@@ -628,7 +628,7 @@
           "type": "tabular"
         },
         {
-          "name": "out_fastp\u0125ase",
+          "name": "out_fastpĥase",
```

Comment:
is this non ascii character on purpose? `out_fastpĥase` There are 2 others.

Reply:
I came across this while re-dumping the json, but changed it in a later commit back to its original

Reply:
yeah that's correct for that version: https://github.com/galaxyproject/tools-iuc/commit/0cdd1b2639fc9c834ba0804dcafb16391006ff6b